### PR TITLE
Update the status page endpoint to include realtime

### PR DESCRIPTION
**Pull request by jonathan-best**

### CODE REVIEW

File: src/server/routers/status-routes.js [hunk 1]
```diff
@@ -1,26 +1,66 @@
...
+        if (a < b)
+          return 1
+        if (a > b)
+          return -1
+        return 0
+      }).shift();
+
+    let status = 'ok'
+
+    if (moment() - moment(latestIndex, 'YYYY-MM-DD-hh') > ONE_HOUR){
```
Comment (radiodario) [inline src/server/routers/status-routes.js]: shouldn't this be TIME_LIMIT?

Comment (jonathan-best) [inline src/server/routers/status-routes.js]: Because this ones for realtime i used hour so it would warn us if were were an hour behind

**Comment by katsuraku** [comment]
think it looks :+1:

**Comment by jonathan-best** [comment]
Will merge on Monday as a change to dashing will be required

**Pull request merged**